tensorflow/tensorflow.mk

```diff
@@ -1,5 +1,12 @@
 # Makefile for tensorflow plugin for MLDB
 
+ifeq ($(WITH_CUDA),1)
+NUM_CUDA_GPUS:=$(shell nvidia-smi -L | grep 'GPU ' | wc -l)
+else
+NUM_CUDA_GPUS:=0
+endif
+MANUAL_IF_NO_GPUS:=$(if $(call seq,0,$(NUM_CUDA_GPUS)),manual)
```

Comment:
It is not needed, but I like the idea of being able to disable a test if we do not have cuda or if we don't have any GPUs.
So I left it in.